packages/salesforcedx-vscode-apex-replay-debugger/test/unit/activation/getDialogStartingPath.test.ts

```diff
@@ -0,0 +1,49 @@
+import { LAST_OPENED_LOG_FOLDER_KEY } from '@salesforce/salesforcedx-apex-replay-debugger/src';
+import { workspaceUtils } from '@salesforce/salesforcedx-utils-vscode';
+import * as vscode from 'vscode';
+import { dialogStartingPath } from '../../../src/activation/getDialogStartingPath';
+
+describe('getDialogStartingPath', () => {
+  let hasRootWorkspaceStub: jest.SpyInstance;
+  let getLastOpenedLogFolderMock: jest.SpyInstance;
+  let folderExistsMock: jest.SpyInstance;
+  let getUriForMock: jest.SpyInstance;
+  const testPath = '/here/is/a/fake/path/to/';
+
+  beforeEach(() => {
+    hasRootWorkspaceStub = jest.spyOn(workspaceUtils, 'hasRootWorkspace');
+    getLastOpenedLogFolderMock = jest.spyOn(
```

Comment:
Still trying to get these supporting functions from the dialogStartingPath module to be mocked correctly.  These don't seem to be working as written here.